ush/load_ufsda_modules.sh

```diff
@@ -53,6 +53,14 @@ elif [[ -d /scratch1 ]] ; then
   module load prod_util
 elif [[ -d /work ]] ; then
   # We are on MSU Orion
+  # prod_util stuff, find a better solution later...
```

Comment:
Any estimate from the hpc-stack team when this problem will be resolved?   It would be nice to remove this hack sooner rather than later.

Reply:
agreed, it's not likely to happen until the UFS spack-stack and the JEDI spack-stack are one and the same...

Reply:
OK.  Thanks for the timing sequence.

Reply:
The stack unification will take a while till the GSI and GDASApp can be built w/ the same compiler version.